Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3
+
+WORKDIR /usr/src/app
+
+RUN pip install dependency-injector
+RUN pip install dnslib
+RUN pip install sanic
+RUN pip install dependency-injector[yaml]
```

Comment:
Project is using Pipfile to keep it dependencies so it should be enough to use `pipenv` to install them.
1) Ensure docker is using Python 3.8
2) Install pipenv via `pip install pipenv`
3) Copy Pipfile into docker root
4) Run `pipenv install --deploy --system`  (it will install deps in system env instead of creating venv)

Reply:
Sorry, in fact it should be both Pipfile and Pipfile.lock for it to work correctly

Reply:
Done